pyproject.toml

```diff
@@ -25,7 +25,7 @@ classifiers = [
 dependencies = [
     "Jinja2>=3.0.1",
     "PyYAML>=5.0",
-    "aiohttp",
+    "aiohttp>=3.9.1",
```

Comment:
Older versions of `aiohttp`(e.g. 3.8.6) don't expose the `ClientSession.max_field_size` kwarg.